src/genie/libs/parser/iosxe/tests/test_show_ip_nbar_version.py

```diff
@@ -0,0 +1,75 @@
+# Python 
```

Comment:
we already moved to new style unittest by using pyATS. Please check below section in doc.
https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html#test-your-new-parser

and then please add output and expected files like below for this new parser.
```
ShowClock
└── cli
    ├── empty
    │   └── empty_output_output.txt
    └── equal
        ├── golden_output_expected.py
        └── golden_output_output.txt

Reply:
Fixed!